src/index.js

```diff
@@ -0,0 +1,15 @@
+require('bulma')
```

Comment:
[REQUIRED] Remember to use ES6 syntax

![image](https://user-images.githubusercontent.com/73254040/96965222-66c77480-14d1-11eb-8db8-49a3c882e40d.png)

[source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)